ambry-cloud/src/main/java/com/github/ambry/cloud/CloudBlobStore.java

```diff
@@ -637,6 +638,11 @@ public void delete(List<MessageInfo> infos) throws StoreException {
             "Delete", partitionId.toPathString());
       }
     } catch (CloudStorageException ex) {
+      if (ex.getStatusCode() == TOO_MANY_REQUESTS) {
```

Comment:
This is based on your recommendation from your document to deal with the vast number of cosmos exceptions coming in. Additionally, my idea is to silence these 429's so that real errors become visible. 429's aren't really harmful. The impact would be slower backups and with DR efforts, that can be identified and rectified if necessary. We can enable debug level when we want to debug. Besides, we can see 429s and other errors in azure portal as a first response. The current logging isn't really adding any value.

Reply:
Wait, based on your investigation, isn't the vast majority of errors 429 ? However, i didn't find many of those in the logs when i simply tailed the logs. The number of 429s in prod is 0 at the moment.

Reply:
I suggest we can add metrics for 429 errors, and if the 429 percentage is less then 5% which mentioned in the wiki, we can just ignore this issue. But if it's above that threshold, we need to think about increasing the RU. Azure has it's own metric but we can't set alert on it.

Reply:
removed